File: baekjoon/js/1764.js
const fs = require("fs");
const buf = fs.readFileSync(0, "utf-8").toString().trim();

const [, ...inputs] = buf.split("\n");

const result = [];
const set = new Set();

for (const name of inputs) {
  if (set.has(name)) {
    result.push(name);
  }

  set.add(name);
}

result.sort();

console.log(result.length);
console.log(result.join("\n"));

File: baekjoon/rust/1764.rs
use std::collections::HashSet;
use std::io::stdin;

fn main() {
    let mut lines = stdin().lines().skip(1);

    let mut result = Vec::new();
    let mut set = HashSet::new();

    while let Some(Ok(line)) = lines.next() {
        let name = line.clone();

        if set.contains(&name) {
            result.push(name.clone());
        }

        set.insert(name);
    }

    result.sort();

    println!("{}", result.len());
    print!("{}", result.join("\n"));
}
